dlpy/model.py

```diff
@@ -2342,30 +2342,114 @@ def save_to_table_with_caslibify(self, path):
 
```

Comment:
did you run the unittests? are they clean? could you please also attach the results of it?

Reply:
test results: https://gist.github.com/yg2288/a6e1bf2a0c6e62256a67c5cb765fef65  

- missing the 'giraffe_dolphin_small' data folder
- seems to be a path error in test_audio_1
- rest are good (I don't have timedata action set, so couldn't run those)